x/poolmanager/keeper_test.go

```diff
@@ -17,7 +17,19 @@ type KeeperTestSuite struct {
 
 const testExpectedPoolId = 3
 
-var testPoolCreationFee = sdk.Coins{sdk.NewInt64Coin(sdk.DefaultBondDenom, 1000_000_000)}
+var (
+	testPoolCreationFee = sdk.Coins{sdk.NewInt64Coin(sdk.DefaultBondDenom, 1000_000_000)}
+	testPoolRoute       = []types.ModuleRoute{
```

Comment:
having CL type here doesn't really matter for test coverage that much here since it's just testing that the genesis updates correctly.

I left it out to minimize merge conflicts because this needs to be backported to v14.x that doesn't have CL type

Reply:
yup makes sense